frontend/src/components/Per-ImplementationPage/PerImplementationPage.tsx

```diff
@@ -0,0 +1,108 @@
+import { Card } from "react-bootstrap";
+import { useLoaderData, Link } from "react-router-dom";
+import { Table, Container } from "react-bootstrap";
+import { ReportData, ImplementationMetadata } from "../../data/parseReportData";
+// @ts-ignore
+import LoadingAnimation from "../../components/LoadingAnimation";
+import DialectCompliance from "./DialectCompliance";
+
+export const PerImplementationPage = () => {
+  const loaderData = useLoaderData() as ReportData;
+
+  const pathSegments = window.location.href.split("/");
+  let implementationName = pathSegments[pathSegments.length - 1];
+
+  let implementationDetail: ImplementationMetadata = {};
+  let allImplementations = {};
+  Object.values(loaderData).map((value) => {
+    allImplementations = {
+      ...allImplementations,
+      ...value.runInfo.implementations,
+    };
+  });
+  Object.keys(allImplementations).forEach((key) => {
+    if (key.includes(implementationName)) {
+      implementationDetail = allImplementations[key];
+    }
+  });
+  return implementationDetail ? (
+    <Container className="p-4">
+      <Card className="mx-auto mb-3 w-75">
+        <Card.Header>Runtime Info</Card.Header>
+        <Card.Body>
+          <Table>
+            <tbody>
+              <tr>
+                <th>Name:</th>
+                <td>{implementationDetail.name}</td>
+              </tr>
+              <tr>
+                <th>Language:</th>
+                <td>
+                  {implementationDetail.language}
```

Comment:
Let's map language to a user friendly name - check out `mapLanguage.ts`

Reply:
Done